Dockerfile.webhook.openshift

```diff
@@ -0,0 +1,8 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.15 AS builder
+WORKDIR /go/src/github.com/kubernetes-csi/external-snapshotter
+COPY . .
+RUN make build
+
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.6:base
```

Comment:
I believe we want `4.7` base images (in both Dockerfiles).

Reply:
Removed.